andela/Lit-Kigali1-Frontend
# #163519095 Setup sass with webpack

#### What does this PR do?
Sets up sass for react components

#### Description of Task to be completed?
- Install sass packages (node-sass, style-loader, css-loader, sass-loader)
- Add sass rules to webpack.config.js
- Add the App.jsx component with basic sass

#### How should this be manually tested?
Clone the repo
Pull this branch
run `npm install`
run `npm run start`
navigate to: `localhost:8080` in your browser you will see the main page

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#163519095

#### Screenshots
N/A

#### Questions:
N/A
